docs/_policies/user_limits.md

```diff
@@ -1,56 +1,71 @@
 # User Limits
+
+## Interactive Compute Limits
+
+The Yens are designed to enable considerable computational usage on our "login" or "interactive" nodes.
+
+As such, we have imposed guidelines for users to follow on these shared systems, depending on the resources of the underlying machine:
```

Comment:
I would maybe add that Jupyter has these limits AND interactive nodes, separately.